src/domains/project/service/project.service.impl.ts

```diff
@@ -121,6 +123,31 @@ export class ProjectServiceImpl implements ProjectService {
     return await this.projectRepository.getProjectsByUserId(input);
   }
 
+  /**
+   * Deletes a member from a project.
+   * @param input - Parameters for identifying the user-project relationship.
+   * @param {string} input.projectId - The id of the project
+   * @param {string} input.userId - The id of the user to delete
+   * @param {string} input.cognitoId - The cognito id of the Project Manager who is trying to delete the member
+   * @returns A Promise that resolves once the member is deleted.
+   * @throws NotFoundException if the user-project role is not found.
+   * @throws ConflictException if the user tries to delete themselves.
+   */
+  async deleteMember({ cognitoId, userId, projectId }: { cognitoId: string; userId: string; projectId: string }): Promise<void> {
```

Comment:
Please, make this service method to return something. It makes easier to test it

Reply:
solved